Assignments/assignment11/src/recursive_operation/first_repeated.rs

```diff
@@ -0,0 +1,21 @@
+use crate::recursive_operation::enum_list::List;
+use crate::recursive_operation::enum_list::List::{Cons, Nil};
+/// This function gives the first repeating element.
+///
```

Comment:
use function name instead This

Reply:
I took the reference from the [https://github.com/knoldus?q=&type=&language=rust]. According to this, the comment is correct. Please visit the same.